velox/dwio/common/Options.h

```diff
@@ -161,7 +161,9 @@ class RowReaderOptions {
   uint64_t skipRows_ = 0;
   std::shared_ptr<UnitLoaderFactory> unitLoaderFactory_;
 
-  TimestampPrecision timestampPrecision_ = TimestampPrecision::kMilliseconds;
+  // After https://github.com/facebookincubator/velox/pull/4680, the precision
+  // could be controlled via HiveConfig.
+  TimestampPrecision timestampPrecision_ = TimestampPrecision::kMicroseconds;
```

Comment:
@rui-mo Is this change intentional? It doesn't seem safe to land as is.

Reply:
Yes, it's not safe because Presto needs milli while Spark needs micro. We need to land https://github.com/facebookincubator/velox/pull/4680 first as it introduces a config to control the precision. After that we can adjust the precision through it.

Reply:
Updated to add hiveConfigs as a parameter of AggregateFuzzer to allow its configuration in SparkAggregationFuzzerTest.